internal/otkdisk/partition.go

```diff
@@ -0,0 +1,36 @@
+package otkdisk
+
+import (
+	"github.com/osbuild/images/pkg/disk"
+)
+
+// Data contains the full description of the partition table as well as extra
+// options and a PartitionMap for easier access. The data under Const should
+// not be modified by a consumer of this data structure.
+type Data struct {
+	Const Const `json:"const"`
+}
+
+type Const struct {
+	KernelOptsList []string `json:"kernel_opts_list"`
+
+	// PartitionMap is generated for convenient indexing of certain partitions
+	// with predictable names in otk, such as
+	// "filesystem.partition_map.boot.uuid"
+	PartitionMap map[string]Partition `json:"partition_map"`
+
+	// Internal representation of the full partition table.
```

Comment:
Maybe we could add  here
```
	// Internal representation of the full partition table. The representation is internal to the partition tools and should not be used by otk directly. No external API gurantees about the conent
```
or something like this?

Reply:
Updated the docstring.